modules/ROOT/assets/attachments/sg.yaml

```diff
@@ -304,43 +328,26 @@ properties:
-
-              On start-up, Sync Gateway will generate the mobile-specific metadata for all the pre-existing documents in the Couchbase Server bucket. From then on, documents can be inserted on the Server directly (with N1QL or SDKs) or through the Sync Gateway REST API.
-
-              #### Import process
-
-              Import is the name for the process by which non-Sync Gateway writes obtain mobile metadata. Any non-Sync Gateway write is eligible for import. A filter function can be specified using the [databases.foo_db.import_filter](#1.5/databases-foo_db-import_filter) property to only import specific documents.
-
-              The document is first run through the Sync Function to compute read security and routing, with the following specifics/differences:
+              Introduced in Sync Gateway 1.5, this property specifies whether to enable Mobile-Web Data Sync.
 
-              - The import is processed with an admin user context in the Sync Function, similar to writes made through the Sync Gateway Admin API. This means that `requireAccess`, `requireUser` and `requireRole` calls in the Sync Function are treated as no-ops.
-              - During import, `oldDoc` is `nil` when the Sync Function is executed.
-
-              Only one node in a Sync Gateway cluster should be configured for feed-based import - it processes the full DCP stream. The [Import+](#1.5/log) log key can be used to troubleshoot the import process further in the logs.
-
-              ##### Import and Sync Gateway Accelerator
+              This property works in conjunction with the [import_docs](#databases-foo_db-import_docs) property.
 
-              In a deployment with Sync Gateway Accelerator, feed-based import processing is performed by the Accelerator nodes. Import work is sharded across the Accelerator cluster, in the same way that indexing work is sharded today.
+              `enable_shared_bucket_access` needs to be `true` on all nodes participating in such a configuration where as `import_docs` can only be set to `true` on a single node.
 
               #### Tombstones
 
-              When convergence is enabled, mobile tombstones are now also server tombstones. The document body is deleted, and the information required to replicate the tombstone is retained in the mobile extended attribute.
-
-              ##### Metadata Purge Interval
+              When `enable_shared_bucket_access` is enabled, mobile tombstones are now also server tombstones. The document body is deleted, and the information required to replicate the tombstone is retained in the mobile extended attribute.
 
               The server's metadata purge interval becomes an important consideration for mobile deployments under convergence. When the server purges a tombstone (based on the metadata purge interval), that tombstone will no longer be replicated to mobile clients.
```

Comment:
Move the Metadata Purge Interval Section to https://docs.couchbase.com/sync-gateway/2.6/shared-bucket-access.html